models/cow_protocol/ethereum/cow_protocol_ethereum_schema.yml

```
@@ -274,15 +274,15 @@ models:
     columns:
       - *trader
       - *referrer
-  - name: cow_protocol_ethereum_price_improvement
+  - name: cow_protocol_ethereum_trades_slippage
```

Nit: I would very much prefer to call this "trade slippage"
```suggestion
  - name: cow_protocol_ethereum_trade_slippage
```